apps-rendering/src/components/Tags/GalleryTags.tsx

```diff
@@ -0,0 +1,90 @@
+// ----- Imports ----- //
+
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { border } from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import { from, neutral } from '@guardian/source-foundations';
+import { grid } from 'grid/grid';
+import type { Item } from 'item';
+import { getFormat } from 'item';
+import type { FC } from 'react';
+import { defaultStyles, DefaultTags } from './Tags.defaults';
+
+// ----- Component ----- //
+
+const containerStyles = css`
+	${grid.container}
+`;
+
+const leftColStyles = css`
+	position: relative;
+
+	${from.leftCol} {
+		grid-row: 4;
+		${grid.column.left}
+
+		&::before {
+			content: '';
+			position: absolute;
+			top: 0;
+			bottom: 0;
+			border-left: 1px solid ${neutral[20]};
+			left: -10px;
+		}
+	}
+`;
+
+const rightColStyles = css`
+	position: relative;
+
+	${from.leftCol} {
+		grid-row: 4;
+		${grid.column.right}
+
+		&::after {
+			content: '';
+			position: absolute;
+			top: 0;
+			bottom: 0;
+			border-left: 1px solid ${neutral[20]};
```

Comment:
would `border.galleryImage` work here as well?

```suggestion
			border-left: 1px solid ${border.galleryImage(format)};
```